common/components/common/groups/ContactModal.jsx

```diff
@@ -0,0 +1,128 @@
+// @flow
+
+import React from 'react';
+import {Modal, Button, ControlLabel, FormControl, FormGroup} from 'react-bootstrap';
```

Comment:
Webpack warning: `"export 'ControlLabel' was not found in 'react-bootstrap'`

We'll have to update those components to use Form Label instead. ([docs](https://react-bootstrap.github.io/components/forms/))

Reply:
Addressed in 0caef95.